pytopo/rf/alazar_acquisition.py

```diff
@@ -195,6 +195,39 @@ def post_acquire(self):
         return (data.astype(np.float32) / (2**self._nbits)) - 0.5
 
 
+class AvgRecCtl(BaseAcqCtl):
+
```

Comment:
having no docstrings, i assume, is ok because everybody knows what's going on, right?

Reply:
Yes that seems to be the assumption this file is written under. As there are no docstrings in it at the moment

Reply:
thank you very much for the docstrings!!!